code-reviews/example2/main.py

```diff
@@ -0,0 +1,16 @@
+from math_operations import add, multiply, subtract
+
+num1 = 5
+num2 = 3
+
+sumResult = add(num1, num2)
+print("The sum result is: " + sumResult)
+
+product_result = multiply(num1 * num2)
```

Comment:
Looking at the other two functions, I am guessing there should be a comma rather than multiply sign here?